lib/connect-fs.js

```diff
@@ -166,7 +165,7 @@ module.exports = function(session){
     try {
       var maxAge = sess.cookie.maxAge;
       var now = new Date().getTime();
-      var expired = maxAge ? now + maxAge : now + oneDay;
+      var expired = maxAge ? now + maxAge : now + oneDay * 1000;
```

Comment:
@x4base we should set oneDay to 86400000 from start, at line 20, what do you think?